components/ai_chat/resources/untrusted_conversation_frame/components/context_actions_assistant/index.tsx

```diff
@@ -37,6 +42,20 @@ export default function ContextActionsAssistant(
     )
   }
 
+  function handleRegenerateAnswer(selectedModelKey: string) {
+    if (!props.turnUuid) {
+      return
+    }
+    conversationContext.conversationHandler?.regenerateAnswer(
+      props.turnUuid,
+      selectedModelKey
+    )
+  }
+
+  const leoModels = conversationContext.allModels.filter(
+    (model) => model.options.leoModelOptions
+  )
```

Comment:
Interesting - so we can't regenerate answers with custom models?

Reply:
As mentioned in the description, it was hard to submit a request to a specific model in a different type of engine (between BYOM or ConversationAPIClient ones) without changing the model of the conversation given how EngineConsumer classes are done and managed in the current implementation, to make the scope manageable, it is not supported in this PR.